db/data/20200727124151_add_transparency_identifiers_to_all_activities.rb

```diff
@@ -0,0 +1,19 @@
+class AddTransparencyIdentifiersToAllActivities < ActiveRecord::Migration[6.0]
+  def up
+    activities = Activity.all
+    activities.each do |activity|
+      next if activity.transparency_identifier.present?
+      transparency_identifier = activity.iati_identifier
+      activity.update(transparency_identifier: transparency_identifier)
+      activity.save!
+    end
+  end
+
+  def down
```

Comment:
This could be done as a single query with `Activity.update_all(transparency_identifier: nil)`, but this implementation raises an exception if any of them fail, which might be safer. I'd only change this if we have a very large number of activities to update.